src/orders/components/OrderTransactionRefundPage/components/OrderTransactionRefundTable/RefundTableProductCell.tsx

```diff
@@ -9,25 +10,15 @@ interface RefundTableProductCellProps {
 }
 
 export const RefundTableProductCell: React.FC<RefundTableProductCellProps> = ({ line, index }) => {
-  // TODO: Abstract and move it to dedicated hook in MERX-359
-  const productNameRefs = React.useRef<HTMLElement[]>([]);
-  const variantNameRefs = React.useRef<HTMLElement[]>([]);
-  const isOverflowing = (index: number): false | undefined => {
-    const productElement = productNameRefs.current[index];
-    const variantElement = variantNameRefs.current[index];
+  const { getRefForIndex: getProductNameRefForIndex, isOverflowing: isProductOverflowing } =
+    useOverflowDetection<HTMLElement>();
+  const { getRefForIndex: getVariantNameRefForIndex, isOverflowing: isVariantOverflowing } =
+    useOverflowDetection<HTMLElement>();
 
-    if (!productElement || !variantElement) {
-      return undefined;
-    }
-
-    const productOverflow = productElement.scrollWidth > productElement.clientWidth;
-    const variantOverflow = variantElement.scrollWidth > variantElement.clientWidth;
-
-    return productOverflow || variantOverflow ? undefined : false;
-  };
+  const isAnyOverflowing = isProductOverflowing(index) || isVariantOverflowing(index);
 
   return (
-    <Tooltip open={isOverflowing(index)}>
+    <Tooltip open={isAnyOverflowing ? undefined : false}>
```

Comment:
open is boolean so why not: open={isAnyOverflowing} ?

Reply:
Setting open to `true` makes the tooltip always visible, even when not hovering. `undefined` displays it only on hover and `false` disables it